packages/next/client/index.js

```diff
@@ -76,9 +76,11 @@ export const emitter = mitt()
 export default async ({
   webpackHMR: passedWebpackHMR
 } = {}) => {
-  // This makes sure this specific line is removed in production
+  // This makes sure these specific lines are removed in production
   if (process.env.NODE_ENV === 'development') {
     webpackHMR = passedWebpackHMR
+    const initializeBuildWatcher = require('./dev-build-watcher').default
+    initializeBuildWatcher()
```

Comment:
I don't need `webpackHMR` anymore so I can move the initialization somewhere else. I am not sure where would be the best place.